static/src/javascripts/projects/common/modules/experiments/tests/rtrt-email-form-inline-footer.js

```diff
@@ -56,29 +58,29 @@ define([
         };
 
         this.id = 'RtrtEmailFormInlineFooter';
```

Comment:
just be sure that keeping the test name the same won't mess things up - if it's already put it in people's localstorage you might end up with unbalanced sample sizes